### PR TITLE
feat: use docker compose instead of docker-compose

### LINKED ISSUE #17: switch from docker-compose to docker compose
Docker Compose is now in the Docker CLI, try `docker compose up`

### PR DESCRIPTION
**Pull request by Ruesa18**
fixes: #17

### CODE REVIEW

**Comment by Ruesa18** [comment]
I will use this in the next few weeks as a test if it works ok :)

**Comment by renestalder** [comment]
Works flawlessly btw.

When I was setting up Fedora Silverblue on a machine and dde, I wondered, if `docker-compose` is still valid in a certain scenario. There are still two ways of installing composer: Either as a docker plugin (`docker-compose-plugin`), which maches the command `docker compose` are as a separate package (`docker-compose`), where the command stays as `docker-compose`. Therefore, the question for me is, if the whole dde setup should be smart enought to check which of both commands are available? Or can we be sure that `docker-compose` is going away?

**Comment by Ruesa18** [comment]
Yeah, I can approve @renestalder's message. For me it has worked flawlessly as well since I changed it.
True, maybe a check would be a good idea. @rugbymauri do you think we could check that on installation and then preset the config? I think it would benefit us if the user could still switch it later or do you see that different? :thinking:

**Comment by renestalder** [comment]
So, here is the stretch idea, as from my testing of other (dde unrelated) docker compose setups in podman-remote/podman-compose/docker-compose environments:

- Allow to set the binary command manually in an `.env` or similar (in case you have multiple environments installed and need to control it explicitly).
- If not set, check which command exists from the most obvious one to the less obvious one, e.g.  
  `type "docker-compose" > /dev/null`
- Save the command to a variable and run any compose command with this variable.

Just some brainstorming.

e.g. in Make (in case docker is run as make commands. That's what I use in my own projects):

```
COMPOSE_CMD = docker compose

ifeq (,$(type docker-compose &> /dev/null))
	COMPOSE_CMD = docker-compose
endif

setup:
	$(COMPOSE_CMD) run --rm <service> bash -c "<command>"
```

In Bash:

```
if type "docker-compose" > /dev/null; then
  # set variable. Then use variable
fi
```

**Comment by Ruesa18** [comment]
@renestalder interesting idea :thinking: 
This could be really helpful for some special projects where you - for some reason - depend on a specific container engine.
Shall we make a separate issue for this and discuss it further over there?

**Comment by renestalder** [comment]
> @renestalder interesting idea thinking This could be really helpful for some special projects where you - for some reason - depend on a specific container engine. Shall we make a separate issue for this and discuss it further over there?

Sure that. Let's put that in a separate proposal.

File: README.md [hunk 1]
```diff
@@ -26,7 +26,6 @@ Features include:
 
 * macOS, Ubuntu or Windows (WSL 2)
 * [Docker 17.09.0+](https://docs.docker.com/)
```
Comment (s4mpl3d) [inline README.md:28]: @Ruesa18 this version does not work with `docker compose`

**Review by s4mpl3d — CHANGES_REQUESTED** [review]
@Ruesa18 can you please rebase the pr?

**Comment by s4mpl3d** [comment]
Correction: resolved with #54

**Pull request closed without merging**